x-pack/plugins/ml/server/models/data_visualizer/data_visualizer.ts

```diff
@@ -612,7 +626,9 @@ export class DataVisualizer {
     // Value count aggregation faster way of checking if field exists than using
     // filter aggregation with exists query.
     const aggs: Aggs = datafeedAggregations !== undefined ? { ...datafeedAggregations } : {};
-    const runtimeMappings: { runtime_mappings?: RuntimeMappings } = {};
+    const runtimeMappings: { runtime_mappings?: RuntimeMappings } = runtimeFields
```

Comment:
it would be good to use a `isPopulatedObject` check here too.

Reply:
Updated here [`1e708e1` (#95252)](https://github.com/elastic/kibana/pull/95252/commits/1e708e1ecac866e112b94fc94bd123d69cdfa582)